x/gamm/keeper/migrate.go

```diff
@@ -109,27 +109,46 @@ func (k Keeper) GetLinkedBalancerPoolID(ctx sdk.Context, concentratedPoolId uint
 	return sdk.BigEndianToUint64(balancerPoolIdBigEndian), nil
 }
 
-// OverwriteMigrationRecords sets the balancer to gamm pool migration info to the store
-// Deletes all existing records, migrationInfo in state is completely overwitten by the given migrationInfo.
-func (k Keeper) OverwriteMigrationRecords(ctx sdk.Context, migrationInfo types.MigrationRecords) {
+// deleteMigrationKeys deletes all migration records with the given prefixKey.
+func (k Keeper) deleteMigrationKeys(ctx sdk.Context, prefixKey []byte) {
```

Comment:
I guess this is a general `deleteKeys` function right? Thoughts on moving this to osmoutils `store_helper.go` (probably in separate PR)?

Reply:
added this as a helper method in osmoutils https://github.com/osmosis-labs/osmosis/pull/5400/commits/ad924fbdd9b0ef0251dd7bebb97b2ca537bc65bf